consensus/src/threshold_simplex/mod.rs

```diff
@@ -3276,8 +3295,7 @@ mod tests {
             }
 
             // Prepare TLE test data
-            let target = 10u64; // Encrypt for view 10
-            let target_bytes = target.to_be_bytes();
+            let target = Round::new(333, 10); // Encrypt for view 10
```

Comment:
This isn't at all related to your work but will be interesting to ponder how to adapt an application to this (when epoch duration isn't denoted in views may mean some view never gets produced)